plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -409,7 +409,7 @@ def deploy_group_for_pod(pod, release)
     # verify with a temp release so we can verify everything before creating a real release
     # and having to wait for docker build to finish
     def verify_kubernetes_templates!
-      release = Kubernetes::Release.new(project: @job.project, git_sha: @job.commit)
+      release = Kubernetes::Release.new(project: @job.project, git_sha: @job.commit, git_ref: 'master')
```

Comment:
You're adding `git_ref` as a param to this method, but I don't see it used in the method body. Is there some ruby magic going on here I don't understand?